src/test/java/seedu/address/model/tag/ModuleTagTest.java

```diff
@@ -11,10 +11,12 @@ public class ModuleTagTest {
     private static final String MODULE_TAG_STRING = "CS2103T";
     private static final ModuleTag MODULE_TAG = new ModuleTag(MODULE_TAG_STRING);
 
+    /*
```

Comment:
Gave me problems with ambiguous method calls so I decided to comment this out first.

Reply:
By right, these ought to work, but it’s ok if you leave it to one of us to settle.